project_template/models/project.py

```diff
@@ -0,0 +1,70 @@
+from odoo import models, fields, api
+
+
+class Project(models.Model):
+    _inherit = 'project.project'
+
+    is_template = fields.Boolean(string="Is Template",
+                                 copy=False)
+
+    # CREATE A PROJECT FROM A TEMPLATE AND OPEN THE NEWLY CREATED PROJECT
+    def create_project_from_template(self):
+        if " (TEMPLATE)" in self.name:
+            new_name = self.name.replace(" (TEMPLATE)", " (COPY)")
+
+        new_project = self.copy(default={'name': new_name,
+                                         'active': True,
+                                         'total_planned_hours': 0.0})
+        if new_project.subtask_project_id != new_project.id:
+            new_project.subtask_project_id = new_project.id
+        # Clear Email Alias if it was set in the Template
+        if new_project.alias_name:
+            new_project.alias_name = False

```

Comment:
maybe we can set this on the copy default, at line 15.

Reply:
Good point, I removed lines 20-22 and added this to the copy function. Seems to work as intended. Thanks.